tests/core/config/manual/startupFocusEnd.html

```diff
@@ -0,0 +1,23 @@
+<textarea id="editor1">
+	<div>
```

Comment:
`div` element is not needed here, in most cases `ul`/`ol` will be a direct child of editable element so test should reflect that.